package.json

```diff
@@ -9,6 +9,8 @@
   "scripts": {
     "start": "webpack --mode development",
     "dev": "webpack-dev-server --mode development --open",
+    "sass": "node-sass --include-path scss public/scss/main.scss public/css/style.css",
```

Comment:
We don't have a `public` folder in the project

Reply:
we have it

Reply:
Why are you creating that folder since we don't have `public` folder? Put Sass and css files in `src` folder.

Reply:
I think @rukundoeric you need to create a folder called assets in src folder, to pull those folders(scss,css,img..) **src/assets/scss**

Reply:
I agree with you @EmyRukundo